src/SolidityLauncher.ts

```diff
@@ -87,114 +86,128 @@ import {
   collectInitialVariables,
   collectStatistics,
 } from "./util/collection";
+import Yargs = require("yargs");
+import { RemoveIndex } from "@syntest/core/dist/Configuration";
```

Comment:
We should not refer to the dist folder. This is making the build fail as well

Reply:
Fixed, but the build will keep failing until we release a new beta of the core and update the dependency in this project.